src/core/agent/agent.ts

```diff
@@ -531,6 +531,49 @@ class Agent {
       throw error;
     }
   }
+
+  async completeWipeRecords() {
```

Comment:
This is OK but was there no way we could just wipe the DB directly? Like just truncate the tables